File: 백준/Bronze/2588. 곱셈/곱셈.cs
string num1 = Console.ReadLine();
string num2 = Console.ReadLine();

Console.WriteLine(int.Parse(num1) * int.Parse(num2.Substring(2)));
Console.WriteLine(int.Parse(num1) * int.Parse(num2.Substring(1, 1)));
Console.WriteLine(int.Parse(num1) * int.Parse(num2.Substring(0, 1)));
Console.WriteLine(int.Parse(num1) * int.Parse(num2));

File: 백준/Bronze/2588. 곱셈/곱셈.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        String num1 = sc.nextLine();
        String num2 = sc.nextLine();

        int n1 = Integer.parseInt(num1);

        System.out.println(n1 * Integer.parseInt(num2.substring(2)));
        System.out.println(n1 * Integer.parseInt(num2.substring(1, 2)));
        System.out.println(n1 * Integer.parseInt(num2.substring(0, 1)));
        System.out.println(n1 * Integer.parseInt(num2));

        sc.close();
    }
}
